src/command_modules/azure-cli-reservations/azure/cli/command_modules/reservations/_params.py

```diff
@@ -3,12 +3,17 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
+
 # pylint: disable=line-too-long
-from azure.cli.core.commands import register_cli_argument
-
-register_cli_argument('reservations reservation update', 'applied_scope_type', options_list=('--applied-scope-type', '-t'), required=True)
-register_cli_argument('reservations reservation update', 'applied_scopes', options_list=('--applied-scopes', '-s'))
-register_cli_argument('reservations reservation split', 'quantity_1', options_list=('--quantity-1', '-1'), required=True)
-register_cli_argument('reservations reservation split', 'quantity_2', options_list=('--quantity-2', '-2'), required=True)
-register_cli_argument('reservations reservation merge', 'reservation_id_1', options_list=('--reservation-id-1', '-1'), required=True)
-register_cli_argument('reservations reservation merge', 'reservation_id_2', options_list=('--reservation-id-2', '-2'), required=True)
+def load_arguments(self, _):
+    with self.argument_context('reservations reservation update') as c:
+        c.argument('applied_scope_type', options_list=['--applied-scope-type', '-t'], required=True)
+        c.argument('applied_scopes', options_list=['--applied-scopes', '-s'])
+
+    with self.argument_context('reservations reservation split') as c:
+        c.argument('quantity_1', options_list=['--quantity-1', '-1'], required=True)
+        c.argument('quantity_2', options_list=['--quantity-2', '-2'], required=True)
+
+    with self.argument_context('reservations reservation merge') as c:
+        c.argument('reservation_id_1', options_list=['--reservation-id-1', '-1'], required=True)
+        c.argument('reservation_id_2', options_list=['--reservation-id-2', '-2'], required=True)
```

Comment:
Are all of these `required=True` statements needed?  Try removing them and see if they still appear as required.